tox.ini

```diff
@@ -1,22 +1,34 @@
 [tox]
-minversion = 1.6
+minversion = 3.8.0
+envlist = linters,unit,coverage
 skipsdist = True
-envlist = pep8
+ignore_basepython_conflict = True
+skip_missing_interpreters = False
+requires =
+    tox-extra; python_version >= '3.8'
 
 [testenv]
-basepython = python3
-usedevelop = False
-install_command = pip install {opts} {packages}
-
-[testenv:run]
```

Comment:
Please lets keep the run env, I use it sometimes just to execute some znoyder command without venv

Reply:
Sure, I'll add it back